automation/taskcluster/decision_task.py

```diff
@@ -116,24 +117,25 @@ def nightly(track):
 
     subparsers.add_parser('pull-request')
     subparsers.add_parser('push')
-    release_parser = subparsers.add_parser('release')
 
-    release_parser.add_argument('--nightly', action="store_true", default=False)
-    release_parser.add_argument(
-        '--track', action="store", choices=['nightly', 'staging-nightly'], required=True
-    )
+    nightly_parser = subparsers.add_parser('nightly')
+    nightly_parser.add_argument('--staging', action='store_true')
 
-    result = parser.parse_args()
+    release_parser = subparsers.add_parser('release')
+    release_parser.add_argument('tag')
```

Comment:
I need to confirm the tag format with the #fenix team, then I can happily parse it and use it here :)
This could've used a "#todo" marker to make my intentions more clear, :volleyball:

Reply:
Resolving this conversation since `tag` is not used at https://github.com/mozilla-mobile/fenix/pull/1893/files#diff-168d551d2c0830884a5caeb50ae576e9R163